osg-pilot-container/osgpilot_meter

```diff
@@ -440,6 +459,30 @@ def detect_new_jobs(sqldb, current_ts):
     DebugPrint(INFO, "- deleted %s rows from inbox" % rc2)
 
 
+def detect_and_update_partial_jobs(sqldb):
+    DebugPrint(INFO, "Reporting and breaking up partial jobs")
+
+    insert_sql = """
+        insert into outbox (%s)
+        select *
+          from cutoff
+    """ % ", ".join(SEND_ATTRS)
+
+    # reset aggregates
+    update_sql = """
+        update jobs
+           set DaemonStartTime = last_updated,
+               total_cpu = 0,
+               MemoryUsage = 0
+         where last_updated - DaemonStartTime >= {partial_record_cutoff};
+    """.format(partial_record_cutoff=PARTIAL_RECORD_CUTOFF)
+
+    rc1 = sqldb.execute(insert_sql, (dead_cutoff,)).rowcount
+    rc2 = sqldb.execute(update_sql, (dead_cutoff,)).rowcount
+
+    DebugPrint(INFO, "- inserted %s rows into outbox" % rc1)
```

Comment:
Do we make the assumption that the row count for these two queries are always the same?  If so, we should log an error if they are different.

Reply:
Eh, i guess there is no logic that depends on these two row counts being the same.  But the concept is that the same rows are both copied and then updated, so they ought to match.  They do use the same where clause on the same table while in a transaction.  If it makes you more comfortable to see an error in the case that these don't match, the sure i can add a check for that.

Reply:
Added a row count check with error log message per Brian's suggestion